go.mod

```diff
@@ -26,8 +26,8 @@ require (
 )
 
 replace (
-	github.com/dolthub/go-mysql-server v0.18.2-0.20241106010546-3281d09c1f15 => github.com/fanyang01/go-mysql-server v0.0.0-20241107083514-e7c2ac20ddba
-	github.com/dolthub/vitess v0.0.0-20241104125316-860772ba6683 => github.com/apecloud/dolt-vitess v0.0.0-20241107081545-d894da3857d8
+	github.com/dolthub/go-mysql-server v0.18.2-0.20241106010546-3281d09c1f15 => github.com/apecloud/go-mysql-server v0.0.0-20241112031328-30cddba3eea7
```

Comment:
Update the origin version as well: `go get -v github.com/dolthub/go-mysql-server@main` to get the version string.

Reply:
fixed